apps/documentation/src/pages/howToContribute/index.tsx

```diff
@@ -76,7 +81,29 @@ const HowToContribute = (): JSX.Element => (
                 dark: useBaseUrl('/img/newRuleIndexDark.svg'),
               }}
             />
-            <p>Yoooo</p>
+            <p>
+              <ul>
+                <li>Find a name and an error message for your rule</li>
+                <li>Set the categories and the AWS service related to it</li>
+                <li>
+                  Fix a level for the rule, it should reflect the importance and
+                  the effort it takes to be solved. (You can ask for help from
+                  the sls-mentor team to find the right level )
+                </li>
+                <li>
+                  Fetch the configuration of the ressources concerned by the
```

Comment:
top les petits checks mais on peut peut être plus séparer la partie config de la règle de la partie développement de la règle en elle même (à partir de « fetch the configuration of the resources »)

Reply:
<img width="1059" alt="image" src="https://github.com/sls-mentor/sls-mentor/assets/123945453/3b040784-de48-4008-a23c-da79f92ca938">

Reply:
on trouve le titre verification de la colonne de droite pas hyper clair, autre proposition: colonne de gauche: configuration of the rule (ou configuration tout court) / colonne de droite: rule

Reply:
Configuration of the rule / Source code of the rule